docs/astro/sql/operators/transform.rst

```diff
@@ -0,0 +1,45 @@
+==================
```

Comment:
@sunank200 Can we also add a section about the Templating and how it works and the context available while creating templates?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/628/commits/0616e7133112cb5d709783ae113f8ba31cddc5cf)